lms/services/exceptions.py

```diff
@@ -181,7 +187,7 @@ def raise_from(cls, cause, request, response, validation_errors=None):
         error_description = response_json.get("error_description", "")
 
         if {"message": "Invalid access token."} in errors:
-            raise OAuth2TokenError(**kwargs) from cause
+            raise OAuth2TokenError(refreshable=True, **kwargs) from cause
```

Comment:
This only marks errors as refreshable when we get this specific `{"message": "Invalid access token."}` response body from Canvas.

We don't want `refreshable=T